scripts/yourface.coffee

```diff
@@ -0,0 +1,40 @@
+# Description
+#   Your face is a hubot script. It replies with "your face"isms.
+#
+# Configuration:
+#   HUBOT_YOURFACE_PERCENT (optional)
+#		Percent chance that hubot will repond with a "Your face". Default is 40%
```

Comment:
Is that being set somewhere? It looks like it will just default to 40 if the env isn't defined